dask_bigquery/core.py

```diff
@@ -0,0 +1,260 @@
+from __future__ import annotations
+
+import logging
+from collections.abc import Iterable
+from contextlib import contextmanager
+from functools import partial
+
+import pandas as pd
+import pyarrow
+from dask.base import tokenize
+from dask.dataframe.core import new_dd_object
+from dask.highlevelgraph import HighLevelGraph
+from dask.layers import DataFrameIOLayer
+from google.cloud import bigquery, bigquery_storage
+
+
+@contextmanager
+def bigquery_client(project_id=None, with_storage_api=False):
```

Comment:
For what I see every time we use the created context manager we use it with `with_storage_api=True` which enables the use of the storage API. My understanding is that we want to use this based on these two comments https://github.com/dask/dask/issues/3121#issuecomment-543276620 and https://github.com/dask/dask/issues/3121#issuecomment-577843296

Reply:
That makes sense. I'm suggesting that since we never use `with_storage_api=False`, we remove it as an option from our custom `bigquery_client` context manager. We can always add it back in the future if needed, but right now it's just an unnecessary keyword argument (since we always call it with `with_storage_api=True`)